### ISSUE TITLE
Hyperterm 0.2.1 cursor issue

### ISSUE THREAD

**Issue by montogeek** [zeit/hyper#31]
Hi!

In the last update the cursor has an issue, it's not aligned with the _input line_:
![screen shot 2016-07-03 at 20 17 33](https://cloud.githubusercontent.com/assets/1002461/16548744/8471c646-415b-11e6-9b7f-a0f58e7530f3.png)
![screen shot 2016-07-03 at 20 19 50](https://cloud.githubusercontent.com/assets/1002461/16548743/84718758-415b-11e6-80be-8f1350a821cd.png)

Also, there is a scrollbar by default

**Comment by WoodyWoodsta** [comment]
Having the same issue with the scrollbar, and cursor alignment is way out for me!
![image](https://cloud.githubusercontent.com/assets/8268040/16552473/9195f7dc-41c1-11e6-80c2-27b951b2b7c1.png)

**Comment by rauchg** [comment]
@montogeek @WoodyWoodsta unless I'm missing something, we'll have to dig into hterm :)

**Comment by rauchg** [comment]
Please retry with `0.3.1`.

**Comment by montogeek** [comment]
@rauchg It didn't improved. You don't have this issue?
![screen shot 2016-07-05 at 16 37 17](https://cloud.githubusercontent.com/assets/1002461/16601023/c8f4a55c-42cf-11e6-85ef-7b13a49fbf1c.png)

**Comment by rauchg** [comment]
Not at all. It seems like a character width miscalculation issue that results in overflow. Can you try removing the emojis ?

**Comment by rauchg** [comment]
@WoodyWoodsta's issue has to do with line calculation but in his case is lack of monospace font. For some reason he doesn't have `Menlo` (I'm very very curious why), but we should be falling back to another font now and his issue is probably gone.

**Comment by WoodyWoodsta** [comment]
@rauchg Alignment fixed for me:
![image](https://cloud.githubusercontent.com/assets/8268040/16608431/05beb2a8-434c-11e6-8513-6c85a5d415b7.png)

**Comment by montogeek** [comment]
@rauchg Removing the emojis fixed the issue

**Comment by nwwells** [comment]
I get the same problem with a font with ligatures (my example is [Meslo for Powerline](https://github.com/powerline/fonts/tree/master/Meslo). It has an 'fi' ligature:

![image](https://cloud.githubusercontent.com/assets/1039473/17018653/c29cb4ae-4f06-11e6-9ace-c11c69a91d52.png)

Atom had this same problem, which they solved by measuring (unfortunate, but necessary, it seems): http://blog.atom.io/2015/10/29/atom-1-1-is-out.html#new-approach-to-character-measurement

**Comment by marcbachmann** [comment]
There's a css property to disable ligatures. That completely disables ligatures. I guess that's safe to use in latin languages. Not sure about other ones.
```
    termCSS: `
      * {
        font-feature-settings: 'liga' 0;
      }
    `,
```

**Comment by Huvik** [comment]
Hey, i found same problem with Operator Mono and some oh-my-zsh themes. With Menlo font everything works correctly, but Operator Mono is opposite. 
![screen shot 2016-07-29 at 14 52 17](https://cloud.githubusercontent.com/assets/5691926/17248978/0d228d12-559e-11e6-86ca-adbb5d379894.png)

**Comment by montogeek** [comment]
Fixed in Hyper 1.0.0
![image](https://cloud.githubusercontent.com/assets/1002461/21107842/11cc9442-c061-11e6-9949-23047c929a00.png)
